src/api.ts

```
@@ -9,8 +8,9 @@ function wait(delay: number) {
 }
 
 export async function getPeople(): Promise<Person[]> {
-  // keep this delay for testing purpose
-  return wait(500)
-    .then(() => fetch(API_URL))
-    .then(response => response.json());
+  const response = await fetch(API_URL);
+
+  await wait(300);
+
+  return response.json();
```

This line could cause an unexpected error if the network request fails (e.g., returns a 404 or 500 status code). The `fetch` function only rejects on network failure, not on HTTP error codes.

Before calling `response.json()`, you should check if `response.ok` is true. If it's not, you should throw an error to be caught by the calling function. This ensures you handle server-side errors gracefully.